war_or_peace_runner.rb

```diff
@@ -0,0 +1,35 @@
+require "./lib/game.rb"
+
+game = Game.new
```

Comment:
In your `Game` classes initialize method, you are expecting an argument of `(turn)` but you never pass that in here when initializing a new `Game` object. I'm not sure if passing in a `Turn` object was the intention or not but just something I wanted to point out. 

When I tried running `ruby war_or_peace_runner.rb` I was getting `game.rb:9:in 'initialize': wrong number of arguments (given 0, expected 1) (ArgumentError)`.